test/travis/check_all_examples_use_standard_init.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+FBASE_SNIPPET=test/travis/firebase_init.cc.snip
+FBASE_NOWIFI_SNIPPET=test/travis/no_firebase_init.cc.snip
+for example in `find examples/ -name *.ino`;
+do
+  echo $example;
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_SNIPPET | tr -d '\n'`) ||
+  (xxd -p $example | tr -d '\n' | grep -q `xxd -p $FBASE_NOWIFI_SNIPPET | tr -d '\n'`) ||
```

Comment:
Switched the naming.

There may be a better way, I didn't put much time into it. Basically I wanted to be able to match part of the file, including line breaks, to another file. It seemed easiest to just compare them as hexdumps, most text comparison commands (like grep) don't handle line breaks well.